server/models/pattern/utils/utils.go

```diff
@@ -100,3 +102,13 @@ func ToMapStringInterface(mp interface{}) map[string]interface{} {
 	}
 	return res
 }
+
+func GetRandomAlphabetsOfDigit(length int) (s string) {
+	charSet := "abcdedfghijklmnopqrstuvwxyz"
+	for i := 0; i < length; i++ {
```

Comment:
Isn't it there already? When the name of two services have conflict with each other the random string concatenated at the last.

Reply:
Yes, I have just moved this func here.